examples/testmodule/model/_init.cf

```diff
@@ -32,3 +32,14 @@ end
 index ResourceBadIdAttribute(agent, name)
 
 implement ResourceBadIdAttribute using std::none
+
+entity Fail extends std::PurgeableResource:
+    string name
+    string agent
+    string key
+    string value
+end
```

Comment:
It might never get used by anyone else ever, but it would still be nice to document what it does and what the attributes are